velox/row/UnsafeRowFast.cpp

```diff
@@ -1083,9 +1083,12 @@ RowVectorPtr deserializeRows(
       std::vector<char*> nestedData(numRows);
       std::vector<size_t> nestedOffsets(numRows, 0);
       for (auto row = 0; row < numRows; ++row) {
-        const auto offset =
-            readInt32(data[row] + offsets[row] + sizeof(int32_t));
-        nestedData[row] = data[row] + offset;
+        const auto isTopLevelNull = rawNulls && bits::isBitNull(rawNulls, row);
```

Comment:
Would you add a test case that specifically targets this scenario? The fuzzer test is useful, but it doesn't guarantee that this path is hit on every run.